benchmarks/imagenet/resnet50/knn_eval.py

```diff
@@ -56,15 +57,22 @@ def knn_eval(
         num_classes=num_classes,
         feature_dtype=torch.float16,
     )
+    metric_callback = MetricCallback()
     trainer = Trainer(
         max_epochs=1,
         accelerator=accelerator,
         devices=devices,
         logger=TensorBoardLogger(save_dir=str(log_dir), name="knn_eval"),
+        callbacks=[
+            DeviceStatsMonitor(),
```

Comment:
Added `DeviceStatsMonitor` as we also use it for pretraining and linear evaluation.